webcompat/templates/contributors/sub-nav.html

```diff
@@ -5,59 +5,59 @@
     </a>
   </li>
 
-  <li class="sub-nav-item {% if request.url_rule.endpoint in ['contributors_report_bug', 'contributors_reproduce_bug', 'contributors_diagnose_bug', 'contributors_site_outreach'] %} active {% endif %}">
+  <li class="sub-nav-item {% if 'contributors_bug_' in request.url_rule.endpoint %} active {% endif %}">
```

Comment:
@miketaylr first of all. Thanks. This is working.

But I was feeling a bit itchy to make this dependent on the function names instead of the routes.

`request.url_rule.rule` returns the route instead of the function name returned by `request.url_rule.endpoint`

```python
<li class="sub-nav-item {% if  request.url_rule.rule.startswith('/contributors/') %} active {% endif %}">
    <a class="link-text" href="/contributors/report-bug">
      Working with bugs <span class="nested-sub"></span>
    </a>
  ```

Reply:
@karlcow the one area where this becomes less desirable, IMO, is that we can't nicely create groups without changing route names (which we don't want to do, because we have to support redirects forever).

Right now there's 2 "groups", beyond the root `/contributors` route:

"Bug" group

| (proposed) method_name  | route |
| ------------- | ------------- |
| contributors_bug_report  | /contributors/report-bug |
| contributors_bug_reproduce | /contributors/reproduce-bug |
| contributors_bug_diagnosis | /contributors/diagnose-bug | 
| contributors_bug_outreach | /contributors/site-outreach |

"Other" group

| (proposed) method_name  | route |
| ------------- | ------------- |
| contributors_other_tools  | /contributors/build-tools |
| contributors_other_research | /contributors/web-platform-research  |
| contributors_other_events |/contributors/organize-webcompat-events| 

So we can simplify a *little*, but not as much as we could if we rename the methods. Any reason to not change the method names? There are no external dependencies (like changing URLs would create).